components/LanguageSelect.jsx

```diff
@@ -19,7 +19,7 @@ export default class LanguageSelect extends Component {
         Show example query output as:<br />
         <select value={this.props.language} onChange={this.changeLanguage}>
           <option value="mysql">MySQL / MariaDB</option>
-          <option value="pg">PostgreSQL</option>
+          <option value="pg">PostgreSQL / Amazon Redshift</option>
```

Comment:
Isn't this wrong, since redshift has its own dialect name? Now this states that they are the same with postgres.